runtime/queries/heex/injections.scm

```diff
@@ -1,15 +1,23 @@
-; HEEx directives can span multiple interpolated lines of Elixir
-(directive [
-  (expression_value) 
-  (partial_expression_value)
-] @injection.content
-  (#set! injection.language "elixir")
-  (#set! injection.combined))  
+; directives are standalone tags like '<%= @x %>'
+;
+; partial_expression_values are elixir code that is part of an expression that
+; spans multiple directive nodes, so they must be combined. For example:
+;     <%= if true do %>
+;       <p>hello, tree-sitter!</p>
+;     <% end %>
+((directive (partial_expression_value) @injection.content)
+ (#set! injection.language "elixir")
+ (#set! injection.include-children)
+ (#set! injection.combined))
 
-; HEEx Elixir expressions are always within a tag or component
-(expression 
-  (expression_value) @injection.content 
-  (#set! injection.language "elixir"))
+; Regular expression_values do not need to be combined
+((directive (expression_value) @injection.content)
+ (#set! injection.language "elixir"))
+
+; expressions live within HTML tags, and do not need to be combined
+;     <link href={ Routes.static_path(..) } />
+((expression (expression_value) @injection.content)
+ (#set! injection.language "elixir"))
```

Comment:
Seems like the last one was unchanged?

Reply:
Looks like it, yes, but should I then also change the formatting of the two directive parts above? This is how it's formatted upstream https://github.com/phoenixframework/tree-sitter-heex/blob/main/queries/injections.scm

Reply:
Don't make unrelated formatting changes, please; these make it harder to review. (And long-term, we want to run a formatter over all of them anyway.)